jsonschema-core.xml

```diff
@@ -200,6 +200,15 @@
                 <t>
                     A JSON document to which a schema is applied is known as an "instance".
                 </t>
+                <t>
+                    JSON Schema is defined over "application/json" or compatible documents,
+                    including media types with the "+json" structured syntax suffix.
+                </t>
+                <t>
+                    Among these, this specification defines the "application/schema-instance+json"
+                    media type which defines handling for fragments in the URI,
+                    and the "schema" media type parameter.
```

Comment:
I know this was already in the spec... but what does this actually mean? =/

Reply:
It is "application/schema+json" that defines how URI fragments are handled. "application/schema-instance+json" makes no further modifications.

Reply:
@Relequestual you (technically) cannot use any media type parameters with `application/json`, nor can you use JSON Pointer fragments (expressly forbidden in RFC 6901 JSON Pointer).  So `application/schema-instance+json` means "just like `application/json` except you can use JSON Pointer fragments and the schema media type parameter."

In practice, everyone ignores these restrictions, but we should provide a formal way to work with them.

@karenetheridge `application/schema+json` and `application/schema-instance+json` are independently defined.  There is an RFC for `+json` but otherwise media types are not related even if they look like it.  We, of course, define them identically for fragment and media type parameter purposes, but they are separate defintions.

Reply:
I think I had just forgotten what a "media type parameter" was, and looked like, and it is in the spec... so yeah I'm done with this thread. Thanks =]